### DISCUSSION TITLE
How to allow vertical scrolling in Daisy UI Dropdown Menu?

### DISCUSSION THREAD

**Discussion by gurbaaz27** [saadeghi/daisyui#1766] [Q&A] [answered]
At the moment I have a drop-down menu that pops up when the user clicks on it. The drop-down currently has 24 items. But the issue is that this takes up the full screen. How do I make my pop-up scrollable, only displaying around, say 5 items at a time?

![image](https://user-images.githubusercontent.com/60743677/233791804-bc0a6213-387f-49dd-8cb8-1b5d8db3647d.png)

**Comment by saadeghi** [comment]
You can set a height for it and use [`overflow-y-scroll`](https://tailwindcss.com/docs/overflow#scrolling-vertically-always)

Let me know if you have any questions

**Reply by gurbaaz27** [reply]
When I set height as `h-80` and `overflow-y-scroll` (like in the code mentioned below), it insteads form a dropdown across x-axis like this. But I want vertical scrolling instead of this horizontal one.

```javascript
return (
    <div className="dropdown" >
      <label tabIndex={0} className="btn pl-1 max-sm:btn-sm max-sm:pl-0.5 m-1 max-sm:m-0" onMouseDown={CheckAndCloseDropDown}>
        <RiArrowDropDownLine
          size={32}
          color='white'
        />
        <p className="capitalize">{title}</p>
      </label>
      <ul tabIndex={0} className="h-80 overflow-y-scroll menu menu-compact dropdown-content mt-3 p-2 shadow bg-base-100 rounded-box w-52">
        {children}
      </ul>
    </div >
  )
```

![image](https://user-images.githubusercontent.com/60743677/234549837-71d7e22a-59f6-4802-b397-4020ac6f9636.png)

**Reply by saadeghi** [reply, marked as answer]
I see. Add this as well: `flex-nowrap`  
Example: 
https://play.tailwindcss.com/aykoiUH7Ku
